### PR TITLE
deployment: Wait for 'default.target'

### PR DESCRIPTION
**Pull request by tinez**
Waiting for 'sshd' to be responsive is sometimes not enough.
We have cases of OST runs failing where 'sshd' is already responsive,
but 'NetworkManager' is still doing some configuration.

Signed-off-by: Marcin Sobczyk <msobczyk@redhat.com>
Change-Id: I5542971d6aef1f0a8eb2ec3d2acd006649c95570

### CODE REVIEW

File: ost_utils/pytest/fixtures/deployment.py [hunk 1]
```diff
@@ -112,8 +114,10 @@ def deploy(
         LOGGER.info("Environment already deployed")
         return
 
-    LOGGER.info("Waiting for SSH on the VMs")
-    ansible_all.wait_for_connection(timeout=120)
+    LOGGER.info("Waiting for VMs to fully boot")
+    assert_utils.true_within_short(
+        ansible_all.shell("systemctl is-active default.target"), allowed_exceptions=[AnsibleExecutionError]
```
Comment (acernek) [inline ost_utils/pytest/fixtures/deployment.py:119]: you have to pass a function (i.e. lamda)

Comment (tinez) [reply ost_utils/pytest/fixtures/deployment.py]: right, wrong copy-paste :)

**Review by acernek — APPROVED** [review]
LGTM, assuming assert was left out on purpose (previous code was not asserted either)

**Comment by michalskrivanek** [comment]
needs also a feedback when it succeeds

**Comment by tinez** [comment]
It's still wrong, will update in a sec

**Comment by michalskrivanek** [comment]
> It's still wrong, will update in a sec

yeah, just noticed too. nevermind about the extra logs/feedback, i'll post something on top, it's bothering me long enough...

**Review by michalskrivanek — APPROVED** [review]
still using ansible, meh, but it hopefully works:)

**Comment by michalskrivanek** [comment]
it works for me manually, we will only get feedback when we use this massively, let's go...

**Pull request merged**